### PR TITLE
Replacing path concatenation with os.path.join

### LINKED ISSUE #3: Replace path separators with os.path.join
Path separators are written as plain strings, such as this:

``` Python
dstfile = basedir + "/" + os.path.basename(imgpath)
```
this is bad practice. They should be written as this:

``` Python
dstfile = os.path.join(basedir, os.path.basename(imgpath))
```

### PR DESCRIPTION
**Pull request by set-killer**
Closes #3 
Please test!

### CODE REVIEW

**Comment by paper-pauper** [comment]
Sorry but did you use a regex or did you find them manually? If the former, could you please share it?

File: io_scene_dae/export_dae.py [hunk 1]
```diff
@@ -173,33 +173,32 @@ def export_image(self, image):
...
                 img_tmp_path = image.filepath
                 if img_tmp_path.endswith((".bmp", ".rgb", ".png", ".jpeg",
                                           ".jpg", ".jp2", ".tga", ".cin",
                                           ".dpx", ".exr", ".hdr", ".tif")):
-                    image.filepath = basedir + "/" + \
-                        os.path.basename(img_tmp_path)
+                    image.filepath = os.path.join(basedir, os.path.basename(img_tmp_path))
                 else:
-                    image.filepath = basedir + "/" + image.name + ".png"
+                    image.filepath = os.path.join(basedir, image.name + ".png")
```
Comment (paper-pauper) [inline io_scene_dae/export_dae.py]: Try to use string formatting for this:

``` Python
image.name + ".png"
```

**Comment by set-killer** [comment]
Fixed it.

Yes, I was searching into the code, not by regex but with different strings. I'm not yet familiar with the code of this project.

**Comment by akien-mga** [comment]
Looks like there are now merge conflicts due to the previous merged PR, sorry about that. Could you rebase your branch to fix those?

**Comment by set-killer** [comment]
Done.

**Pull request merged**